tests/e2e/backup_restore_suite_test.go

```diff
@@ -261,30 +271,39 @@ var _ = Describe("Backup and restore tests", func() {
 	}
 
 	var _ = AfterEach(func(ctx SpecContext) {
+		knownFlake = false
+		logString := strings.Join(accumulatedTestLogs, "\n")
+		CheckIfFlakeOccured(logString, &knownFlake)
+		accumulatedTestLogs = nil
 		tearDownBackupAndRestore(lastBRCase, lastInstallTime, ctx.SpecReport())
 	})
 
 	DescribeTable("Backup and restore applications",
 		func(brCase BackupRestoreCase, expectedErr error) {
+			if CurrentSpecReport().NumAttempts > 1 && !knownFlake {
+				Fail("No known FLAKE found in a subsequent run, marking test as failed.")
```

Comment:
shouldn't be "... in the previous run..."?

Reply:
yes

Reply:
resolved.